### ISSUE TITLE
windows resized after closing ctrlsf window 

### ISSUE THREAD

**Issue by fasterbru** [dyng/ctrlsf.vim#260]
After closing windows remain resized:
![Peek 2019-03-30 10-48](https://user-images.githubusercontent.com/17349169/55273302-6e656e00-52da-11e9-89c7-3ea8abe07290.gif)
![Peek 2019-03-30 10-50](https://user-images.githubusercontent.com/17349169/55273305-79b89980-52da-11e9-8ead-3c7cb37768b4.gif)

- system and environment:

|  field  |                value                 |
|:-------:|:------------------------------------:|
| os | *Archlinux (kernel: 5.0.4-arch1-1-ARCH)* |
| vim | *neovim v0.3.4* |
| backend | *ag version 2.2.0* |
| locale | *en_US.UTF-8* |


- log:

```vim
Tokens: ['nnoremap']                                                                                                                                                                                                                                                        
ParsedResult: {'pattern': 'nnoremap'}                                                                                                                                                                                                                                       
Options: {'pattern': 'nnoremap', '_vimregex': '\V\cnnoremap', '_vimhlregex': {'normal': '\V\c\(\^\d\+:\.\*\)\@<=nnoremap', 'compact': '\V\c\(|\d\+ col \d\+|\.\*\)\@<=nnoremap'}}                                                                                           
ExecCommand: ag -B 2 -A 5  --smart-case --literal -f --noheading --nogroup --nocolor --nobreak -- "nnoremap" "/home/vlad/tmp/ctrlsf.vim"                                                                                                                                    
TimerStarted: id=63                                                                                                                                                                                                                                                         
Searching...                                                                                                                                                                                                                                                                
HighlightRegex: \V\c\(\^\d\+:\.\*\)\@<=nnoremap                                                                                                                                                                                                                             
```

- vimrc:

    ```vim
  let g:ctrlsf_auto_focus = {"at" : "start"}
  let g:ctrlsf_context = '-A 5 -B 2'
  let g:ctrlsf_default_view_mode = 'normal'
  let g:ctrlsf_position = 'bottom'
  let g:ctrlsf_winsize = '40%'
  let g:ctrlsf_indent = 2
  let g:ctrlsf_mapping = {
        \ "split"   : "<C-s>",
        \ "vsplit"  : "<C-v>",
        \ }
    ```
With an empty config works same.

- file:

any

**Comment by dyng** [comment]
ctrlsf will save your window's size before resizing, then restore them after closing itself.

Maybe there is a plugin conflict, please disable others plugins then try again.

**Comment by fasterbru** [comment]
@dyng I tried to use your config and after setting "noequalalways" the result was the same. In my local config i commented all plugins, but didn't change anything.

**Comment by dyng** [comment]
Fixed in #262

**Comment by fasterbru** [comment]
I fix it in #264
